helix-term/src/keymap/default_keymaps.rs

```diff
@@ -0,0 +1,359 @@
+use std::collections::HashMap;
+
+use helix_core::hashmap;
+use super::macros::keymap;
+use super::{Mode, Keymap};
+
+pub fn default_keymaps() -> HashMap<Mode, Keymap> {
```

Comment:
I split this into its own file, as it is large enough by itself and is worth having its own point of reference.

Reply:
Let's name the file `default.rs` and `fn default` since `keymap` is already in the module name. This will impact some of the open PRs though

Reply:
renamed!